src/Malt.jl

```diff
@@ -632,7 +632,31 @@ function interrupt(w::InProcessWorker)
     nothing
 end
 
+"""
+    Malt.requestgc(w::Worker)
 
+Request a garbage collection on the worker `w`. This is a non-blocking call (on the worker).
+"""
+function requestgc(w::Worker)
+    if !isrunning(w)
+        @warn "Tried to gc a worker that has already shut down." summary(w)
+    else
+        remote_eval_wait(Main, w, :(Base.notify(Main._gc_event)))
+    end
+end
+
+"""
+    Malt.autogc(w::Worker)
+
+Initiate the worker auto gc every `ENV["MALT_AUTO_GC_SECONDS"]` or 900 seconds. 
+"""
+function autogc(w::Worker)
```

Comment:
You can call this once, to turn on the auto gc loop, right? And subsequent calls are ignored? I think that's nice API!

We should document this, and maybe a different name would make this more clear. E.g. `start_background_auto_gc` (I love long names but feel free to get creative)